Dockerfile.ubuntu-32

```diff
@@ -33,11 +37,9 @@ RUN cp -a /root/files/${mono_version} /root && \
     wget https://download.mono-project.com/repo/ubuntu/pool/main/c/core-setup/msbuild-libhostfxr_3.0.0.2019.04.16.02.13-0xamarin3+ubuntu1604b1_i386.deb && \
     wget https://download.mono-project.com/repo/ubuntu/pool/main/m/msbuild/msbuild-sdkresolver_16.3+xamarinxplat.2019.08.08.00.55-0xamarin2+ubuntu1604b1_all.deb && \
     wget https://download.mono-project.com/repo/ubuntu/pool/main/n/nuget/nuget_5.2.0.6090.bin-0xamarin1+ubuntu1604b1_all.deb && \
-    dpkg -i --force-all *.deb && \
+    dpkg -i --force-all ./*.deb && \
     sed -i '/Depends.*mono/d' /var/lib/dpkg/status && \
     ln -s /usr/bin/mono /usr/bin/cli && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/ && \
-    rm *.deb
+    rm ./*.deb
```

Comment:
add ./ before globs in case a filename beginning with - is interpreted as an option